hermes-console-vue/src/components/console-alert/ConsoleAlert.vue

```diff
@@ -0,0 +1,23 @@
+<script setup lang="ts">
+  interface ConsoleAlertProps {
```

Comment:
I would rename all props interfaces to just "Props" as props interfaces can't be used anywhere else. We can't event import props from another file, so i think simplifying this would be a good idea

Reply:
I will inline those types as they are not used anywhere else.